addons/account/models/account_journal_dashboard.py

```diff
@@ -605,7 +607,7 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
                 'number_draft': number_draft,
                 'number_waiting': number_waiting,
                 'number_late': number_late,
-                'sum_draft': currency.format(sum_draft * (1 if journal.type == 'sale' else -1)),
+                'sum_draft': currency.format(sum_draft),
```

Comment:
No need to do it here as it is done L517

Reply:
I would add a comment tho, it's not obvious at all why you don't need it here

Except that, LGTM